src/Web/HTML/HTMLAudioElement.purs

```diff
@@ -57,3 +78,9 @@ toParentNode = unsafeCoerce
 
 toEventTarget :: HTMLAudioElement -> EventTarget
 toEventTarget = unsafeCoerce
+
+foreign import create :: Unit -> Effect HTMLAudioElement
```

Comment:
Why isn't this just `create :: Effect HTMLAudioElement`? Why the extra `Unit ->` arg?

Reply:
You're correct that the Unit isn't necessary, but @toastal is following the existing pattern from the bindings from HTMLImageElement.

Reply:
That's why I accepted it here, though the `Unit` is unnecessary. @hdgarrood would you also be open to fixing these functions so that they're just in `Effect` right now?

Reply:
Ah... That makes more sense. Why were they originally `Unit -> Effect result`?

Reply:
Now _that_ I don't know.

Reply:
I would be open to that, yes. I also don’t know why they’re like this. I generally haven’t been very involved with the web libraries, actually.

Reply:
> Ah... That makes more sense. Why were they originally `Unit -> Effect result`?

'tis a mystery to me also, even though I'm probably the one who did it 🤔